cloudlaunchserver/values.yaml

```diff
@@ -4,7 +4,7 @@
 replicaCount: 1
 image:
   repository: cloudve/cloudlaunch-server
-  tag: latest
+  tag: test
```

Comment:
Should this be latest?

Reply:
yeh also the version bumps shouldn't happen, it's just cause I tested it on this branch and it did the bump, i'll force push at the end to only keep the workflow changes

Reply:
This is still test?